### PR TITLE
HUD indicators

### PR DESCRIPTION
**Pull request by dalderliesten**
Added the desired indicators to the HUD, and refactored the HUD class.

### CODE REVIEW

**Comment by wouterz** [comment]
LGTM

File: src/main/java/nl/tudelft/pixelperfect/gui/GameHeadsUpDisplay.java [hunk 1]
```diff
@@ -59,6 +61,20 @@ public GameHeadsUpDisplay(AssetManager passedMan, Node passedGuiNode, int passed
...
    */
   private void setupTextDisplay() {
+    // Set-up the bitmap text needed for HUD display.
+    setUpBitmapText();
+
+    // Attach the elements to the gui nodes.
+    guiNodes.attachChild(captainLog);
+    guiNodes.attachChild(shipHealth);
+    guiNodes.attachChild(teamScore);
+    guiNodes.attachChild(timeLeft);
```
Comment (jessetilro) [inline src/main/java/nl/tudelft/pixelperfect/gui/GameHeadsUpDisplay.java]: Please do this in a loop.

**Comment by jessetilro** [comment]
Apart from my comment on the repeated code, looks good to me.

File: src/main/java/nl/tudelft/pixelperfect/gui/GameHeadsUpDisplay.java (hunk 1 above)
Comment (dalderliesten) [inline src/main/java/nl/tudelft/pixelperfect/gui/GameHeadsUpDisplay.java]: As I have outlined before, this causes nullpointers on refresh due to JMonkeyEngine's setup.  I have pushed the following commit as proof, and will then revert it.

**Comment by wouterz** [comment]
Since you cant prevent this repeated code, merging

**Pull request merged**